tests/dockerConfiguration/verifySrvDataDirectory_test.js

```diff
@@ -4,50 +4,57 @@ const moment = require('moment');
+  await I.verifyCommand('chmod -R 777 $HOME/srvNoData/ || true');
+  await I.verifyCommand(`docker run -v $HOME/srvNoData:/srv -d --restart always --publish 8081:8080 --name pmm-server-srv ${dockerVersion}`);
 };
 
 const runContainerWithPasswordVariable = async (I) => {
-  await I.verifyCommand(`docker run -v $HOME/srvPassword:/srv -d -e GF_SECURITY_ADMIN_PASSWORD=newpass --restart always --publish 8082:80 --name pmm-server-password ${dockerVersion}`);
+  await I.verifyCommand('mkdir $HOME/srvPassword/ || true');
+  await I.verifyCommand('chmod -R 777 $HOME/srvPassword/');
+  await I.verifyCommand(`docker run -v $HOME/srvPassword:/srv -d -e GF_SECURITY_ADMIN_PASSWORD=newpass --restart always --publish 8082:8080 --name pmm-server-password ${dockerVersion}`);
 };
 
 const runContainerWithPasswordVariableUpgrade = async (I) => {
-  await I.verifyCommand('docker run -v $HOME/srvPasswordUpgrade:/srv -d -e GF_SECURITY_ADMIN_PASSWORD=newpass --restart always --publish 8084:80 --name pmm-server-password-upgrade percona/pmm-server:latest');
+  await I.verifyCommand('mkdir $HOME/srvPasswordUpgrade || true');
+  await I.verifyCommand('chmod -R 777 $HOME/srvPasswordUpgrade/');
+  await I.verifyCommand(`docker run -v $HOME/srvPasswordUpgrade:/srv -d -e GF_SECURITY_ADMIN_PASSWORD=newpass --restart always --publish 8089:8080 --name pmm-server-password-upgrade ${dockerVersion}`);
+  I.wait(30);
   await I.verifyCommand('docker exec pmm-server-password-upgrade yum update -y percona-release');
-  await I.verifyCommand('docker exec pmm-server-password-upgrade sed -i\'\' -e \'s^/release/^/experimental/^\' /etc/yum.repos.d/pmm2-server.repo');
+  await I.verifyCommand('docker exec pmm-server-password-upgrade sed -i\'\' -e \'s^/release/^/experimental/^\' /etc/yum.repos.d/pmm3-server.repo');
   await I.verifyCommand('docker exec pmm-server-password-upgrade percona-release enable percona experimental');
   await I.verifyCommand('docker exec pmm-server-password-upgrade yum clean all');
   await I.verifyCommand('docker restart pmm-server-password-upgrade');
 };
 
 const runContainerWithDataContainer = async (I) => {
-  await I.verifyCommand(`docker run -v srvFolder:/srv -d --restart always --publish 8083:80 --name pmm-server-srv ${dockerVersion}`);
+  await I.verifyCommand(`docker run -v srvFolder:/srv -d --restart always --publish 8083:8080 --name pmm-server-empty-data-container ${dockerVersion}`);
 };
 
 const stopAndRemoveContainerWithoutDataContainer = async (I) => {
-  await I.verifyCommand('docker stop pmm-server-srv');
-  await I.verifyCommand('docker rm pmm-server-srv');
+  await I.verifyCommand('docker stop pmm-server-srv || true');
+  await I.verifyCommand('docker rm pmm-server-srv || true');
 };
 
 const stopAndRemoveContainerWithPasswordVariable = async (I) => {
-  await I.verifyCommand('docker stop pmm-server-password');
-  await I.verifyCommand('docker rm pmm-server-password');
+  await I.verifyCommand('docker stop pmm-server-password || true');
```

Comment:
Is it fine to treat it always as it was stopped successfully? (with `|| true`)

Reply:
yes, tests were failing on trying to stop container that was not even started. This should work fine